### PR TITLE
[amazondashbutton] Changed trigger channel type to 'system.rawbutton' to allow the usage of the 'rawbutton-toggle-switch' profile

### PR DESCRIPTION
**Pull request by cweitkamp**
- Changed trigger channel type to `system.rawbutton` to allow the usage of the `rawbutton-toggle-switch` profile

Signed-off-by: Christoph Weitkamp <github@christophweitkamp.de>

### CODE REVIEW

File: addons/binding/org.openhab.binding.amazondashbutton/src/main/java/org/openhab/binding/amazondashbutton/internal/discovery/AmazonDashButtonDiscoveryService.java [hunk 1]
```diff
@@ -32,8 +34,6 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import com.google.common.collect.Sets;
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.amazondashbutton/src/main/java/org/openhab/binding/amazondashbutton/internal/discovery/AmazonDashButtonDiscoveryService.java]: Should / could we remove it from the manifest?

Comment (cweitkamp) [reply addons/binding/org.openhab.binding.amazondashbutton/src/main/java/org/openhab/binding/amazondashbutton/internal/discovery/AmazonDashButtonDiscoveryService.java]: Unfortunately not ... yet ...

https://github.com/openhab/openhab2-addons/blob/533e5f01be4d5395d9dc99b5e0fb3f22410c49f3/addons/binding/org.openhab.binding.amazondashbutton/src/main/java/org/openhab/binding/amazondashbutton/internal/pcap/PcapNetworkInterfaceService.java#L23-L26
https://github.com/openhab/openhab2-addons/blob/533e5f01be4d5395d9dc99b5e0fb3f22410c49f3/addons/binding/org.openhab.binding.amazondashbutton/src/main/java/org/openhab/binding/amazondashbutton/internal/pcap/PcapUtil.java#L14

**Comment by martinvw** [comment]
@cweitkamp I also had this question:

> is this a breaking change, should we mention it somewhere?

**Comment by cweitkamp** [comment]
@martinvw Sry, I missed that one. I am sure, that it is not a breaking change. Most of the people use a rule like the following which will work in the same way afterwards:
 ```java
 rule "My Dash Button pressed"
 when
     Channel "amazondashbutton:dashbutton:fc-a6-67-0c-aa-c7:press" triggered
 then
     logInfo("amazondashbutton", "My Dash Button has been pressed")
 end
```

**Pull request merged**